### DISCUSSION TITLE
Will there be more Docs?

### DISCUSSION THREAD

**Discussion by Brianalmeida** [rcoh/angle-grinder#166] [Q&A]
Hi there!

I'm new to angle-grinder and was curious if there will be more documentation on this tool?

**Comment by rcoh** [comment]
hello! most of the docs are in the readme—there are also a couple of blog posts floating around the web where folks take deeper dives. I'd love to know more about what sort of docs would be helpful, I'm happy to write them

**Comment by Brianalmeida** [comment]
It would be interesting to see how you can use `angle-grinder` combined with `lnav`.
